pkg/apis/garden/v1beta1/types.go

```diff
@@ -898,9 +898,14 @@ type Kube2IAMRole struct {
 // Backup holds information about the backup schedule and maximum.
 type Backup struct {
 	// Schedule defines the cron schedule according to which a backup is taken from etcd.
-	Schedule string `json:"schedule"`
+	// +optional
+	Schedule *string `json:"schedule,omitempty"`
```

Comment:
That's an incompatible change, please keep the previous behavior: https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#on-compatibility.